### PR TITLE
two separate package lists

### PR DESCRIPTION
**Pull request by isuruf**
Checklist
* [x] Pushed the branch to main repo
* [x] CI passed on the branch

Depends on https://github.com/conda-forge/webservices-dispatch-action/pull/60

### CODE REVIEW

File: conda_forge_webservices/update_me.py [hunk 1]
```diff
@@ -103,21 +108,21 @@ def main():
             repo.index.add(pth)
 
             msg_vers = (
-                "Redeploy for package updates:\n\n" +
                 "\n".join(["* `{}={}`".format(k, v) for k, v in to_install.items()])
```
Comment (jakirkham) [inline conda_forge_webservices/update_me.py:111]:
```suggestion
                "Redeploy for package updates:\n\n" +
                "\n".join(["* `{}={}`".format(k, v) for k, v in to_install.items()])

```

File: conda_forge_webservices/update_me.py [hunk 2]
```diff
@@ -103,21 +108,21 @@ def main():
...
             )
 
-            repo.index.commit("redeploy for '%s'" % msg_vers)
+            repo.index.commit("Redeploy for '%s'" % msg_vers)
```
Comment (jakirkham) [inline conda_forge_webservices/update_me.py:114]:
```suggestion
            repo.index.commit(msg_vers)
```

**Review by jakirkham — COMMENTED** [review]
Thanks Isuru! 🙏

Would suggest we build the commit message once

**Review by beckermr — APPROVED** [review]

**Review by jakirkham — APPROVED** [review]

File: conda_forge_webservices/update_me.py [hunk 3]
```diff
@@ -102,22 +107,27 @@ def main():
...
-                "\n".join(["* `{}={}`".format(k, v) for k, v in to_install.items()])
-            )
+            if len(to_install) > 1:
+                msg = (
+                    "Redeploy for package updates\n\n" +
+                    "\n".join(["* `{}={}`".format(k, v) for k, v in to_install.items()])
+                )
+            else:
+                (k, v), = to_install.items()
+                msg = f"Redeploy for package update: {k}={v}"
```
Comment (jakirkham) [inline conda_forge_webservices/update_me.py:117]: Maybe we could use code format for the package version

No strong feelings though

```suggestion
                msg = f"Redeploy for package update: `{k}={v}`"
```

**Review by jakirkham — APPROVED** [review]

**Comment by jakirkham** [comment]
Thanks Isuru and Matt! 🙏

**Pull request merged**